### PR TITLE
chore(ci): fix e2e tests

### PR DESCRIPTION
**Pull request by Haroenv**
- changed examples script to allow building a single example
- actually build & move folders for e-commerce demo
- fix a small error in slider in the example

### CODE REVIEW

File: examples/e-commerce/src/App.vue [hunk 1]
```diff
@@ -302,7 +302,6 @@
               <div
                 slot-scope="{
                   currentRefinement,
-                  nbPages,
```
Comment (yannickcr) [inline examples/e-commerce/src/App.vue:305]: Why removing this? Is it unused?

File: examples/e-commerce/src/App.vue [hunk 2]
```diff
@@ -515,8 +514,8 @@ export default {
     formatNumber,
     toValue(value, range) {
       return [
-        value.min !== null ? value.min : range.min,
-        value.max !== null ? value.max : range.max,
+        value.min != null ? value.min : range.min,
+        value.max != null ? value.max : range.max,
```
Comment (yannickcr) [inline examples/e-commerce/src/App.vue:518]: Would not be better to test for `value.min !== null && value.min !== undefined` or `typeof value.min === 'number'` (if this is the correct condition) here?

File: examples/e-commerce/src/App.vue (hunk 1 above)
Comment (Haroenv) [reply examples/e-commerce/src/App.vue:305]: yep, unused

**Review by eunjae-lee — APPROVED** [review]
The change looks good to me as long as the CI passes :)

**Pull request merged**